react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' }
+];
+
+function ListPokemons(props) {
+  const pokemons = props.pokedex;
+  const listPokemons = pokedex.map(pokemon =>
+    <li>{pokemon.name}</li>
```

Comment:
be sure to give each element a `key` prop. You should not see any errors or warnings in your browser console.

Reply:
it's been revised!